### PR TITLE
Link to all HTML-related tests from html/README.md

### PR DESCRIPTION
**Pull request by foolip**
Part of https://github.com/whatwg/html/issues/3014.

### CODE REVIEW

**Comment by foolip** [comment]
@annevk, FYI, custom-elements is the only thing listed that doesn't have a README of the shape you made in https://github.com/w3c/web-platform-tests/pull/5385. Maybe it should?

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-10 21:05:19
Finished: 2017-09-10 21:11:00





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18358)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/273950742)

**Review by domenic — COMMENTED** [review]
It'd be nice if these were links, like the webappapis/atob above them?

**Comment by foolip** [comment]
Now linked. I don't know if there's much point in the leading slashes, I copied the style from [XMLHttpRequest/README.md](https://github.com/w3c/web-platform-tests/blob/master/XMLHttpRequest/README.md).

**Review by domenic — APPROVED** [review]

File: html/README.md [hunk 1]
```diff
@@ -5,3 +5,15 @@ This directory contains tests for [HTML](https://html.spec.whatwg.org/multipage/
 Sub-directory names should be based on the URL of the corresponding part of the
 multipage-version specification. For example, The URL of
 "8.3 Base64 utility methods" is [https://html.spec.whatwg.org/multipage/webappapis.html#atob](https://html.spec.whatwg.org/multipage/webappapis.html#atob). So the directory in WPT is [webappsapis/atob/](webappsapis/atob).
```
Comment (annevk) [inline html/README.md]: Fix this link to webappapis/atob while here?

**Comment by annevk** [comment]
@foolip yeah, I think custom-elements should have that too. Though it needs to point to both DOM and HTML.

The same applies to `/shadow-dom` btw which you seem to have forgotten about here?

**Comment by zcorpan** [comment]
Added `innerText` in a new commit.

**Comment by foolip** [comment]
@annevk @zcorpan, shadow DOM is part of DOM, not HTML, right?

File: html/README.md (hunk 1 above)
Comment (foolip) [reply html/README.md]: Oh, I didn't notice it was broken, will fix.

**Comment by annevk** [comment]
Both. `<slot>` is in HTML for instance, as are many modifications to how HTML deals with things, e.g., around `<link>` and `<dialog>`.

**Comment by foolip** [comment]
@zcorpan already added shadow-dom too, awesome!

> I think custom-elements should have that too. Though it needs to point to both DOM and HTML.

I haven't done it in this PR, will do it separately.

**Pull request merged**